packages/next/server/response-cache.ts

```diff
@@ -1,22 +1,39 @@
 import { IncrementalCache } from './incremental-cache'
 import RenderResult from './render-result'
 
-interface CachedRedirectValue {
+export interface CachedRedirectValue {
   kind: 'REDIRECT'
   props: Object
 }
 
 interface CachedPageValue {
   kind: 'PAGE'
+  // this needs to be a RenderResult so since renderResponse
+  // expects that type instead of a string
   html: RenderResult
   pageData: Object
 }
 
-export type ResponseCacheValue = CachedRedirectValue | CachedPageValue
+export interface CachedImageValue {
+  kind: 'IMAGE'
+  etag: string
+  buffer: Buffer
+  extension: string
+  isMiss?: boolean
+  isStale?: boolean
+  revalidate: number
```

Comment:
How about calling this `maxAge` to make it match the rest of the code?

Reply:
Looks like we aren't using this now so removed it